core/src/test/java/org/javarosa/core/form/api/test/TextFormTests.java

```diff
@@ -155,9 +92,9 @@ public void testTextForms() {
             fail("getLongText() not falling back to default text form correctly");
```

Comment:
Hey,

These are fixed in the 2.22 branch.

On Mon, Jul 20, 2015 at 2:06 PM, Phillip Mates <notifications@github.com>
wrote:

> In core/src/test/java/org/javarosa/core/form/api/test/TextFormTests.java
> <https://github.com/dimagi/javarosa/pull/109#discussion_r35024534>:
>
> > @@ -155,9 +92,9 @@ public void testTextForms() {
> >              fail("getLongText() not falling back to default text form correctly");
>
> Fails, due to test resource input not matching expected behavior
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/javarosa/pull/109/files#r35024534>.
>